docs/conf.py

```diff
@@ -119,3 +119,163 @@
 # Tell sphinx-autodoc-typehints to generate stub parameter annotations including
 # types, even if the parameters aren't explicitly documented.
 always_document_param_types = True
+
+
+# -- Patch sphinx.ext.autosummary.generate ----------------------------------
+
+# This patch is needed to make autosummary provide the "annotations"
+# variable so we can exclude function attributes from the methods list
+# in flax_module.rst. The patch as such only adds this single line:
+#
+#     ns['annotations'] = list(getattr(obj, '__annotations__', {}).keys())'
+#
+# We should consider sending a PR to sphinx so we can get rid of this.
```

Comment:
Could you add a `TODO` and reference to an open issue assigned to somebody for this?